Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.cpp

```diff
@@ -188,6 +188,15 @@ MacroscopicProperties::InitData ()
     else if (m_mag_Ms_s == "parse_mag_Ms_function"){
         InitializeMacroMultiFabUsingParser(m_mag_Ms_mf.get(), getParser(m_mag_Ms_parser), lev);
     }
+    // if there are regions with Ms=0, the user must provide mur value there
+    if ((*m_mag_Ms_mf).min(1,0) < 0.){
```

Comment:
```suggestion
    if ((*m_mag_Ms_mf).min(0) < 0.){
```
0-based indexing. Also the default is 0 ghost cells.

Reply:
@ajnonaka, what does` *.min(1,0)` do? The 1 is for nodal? and 0 for number of ghost cells?

Reply:
1st argument is which component
2nd argument is number of ghost cells (optional, default=0)